### DISCUSSION TITLE
如何安装使用它？

### DISCUSSION THREAD

**Discussion by liuxyon** [database64128/swgp-go#23] [Q&A]
我没有看到说明如何安装它？ ubuntu and pfsense 系统.

**Comment by database64128** [comment]
Well, it's up to you to figure it out.

**Reply by liuxyon** [reply]
我的意思是下载你的文件后如何安装和运行呢？例如ubuntu 22.04.

**Reply by database64128** [reply]
If you want to "install" it on Ubuntu, you have to do the packaging yourself. Or you can just run the binary like how you normally do.
